src/logging_stacktrace.c

```diff
@@ -127,7 +127,7 @@ void getStackAsString(char* destination, size_t size)
             }
         }
     }
-    destination[0] = '\0';
+    destination[-1] = '\0';
```

Comment:
>-1 [](start = 16, length = 2)

Can you add a task in the things that never get done to write unit tests for this?
Otherwise cheers!

Reply:
https://msazure.visualstudio.com/One/_workitems/edit/8559770

Reply:
every line of code hides a bug...